learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -313,24 +336,26 @@ Likewise, hold *Control* (*Command* on Mac) to drop a setter:
 Signals
 ~~~~~~~
 
-In the *Connecting Signals to a Function* item we have already learned about *Signals*. It is also possible to create your own
-signals in a script and use them. For this, just do the same steps you did for variables in the previous step, except for *Signals*:
+In the *Connecting Signals to a Function* item we have already learned about *Signals*. It is also possible to create your own signals in a script and use them. For this, just do the same steps you did for variables in the previous step, except for *Signals*:
```

Comment:
Don't say "just"

This part could be clearer. Don't assume the person is reading in order, so they might not "have already learned"